### ISSUE TITLE
Middleware double wrapping json

### ISSUE THREAD

**Issue by yong24s** [CS5331-1718-G3/rest-api-development#17]
```
const transformer = function(json) {
  // Don't transform errors.
  if (json && json.status === false) {
    return removeEmpty(json);
  }
  return removeEmpty({ status: true, result: json }); <-- 
};

```
`{"status":true} `may get wrapped by the above code and result in this
` {"status":true,"result":{"status":true}}`

I have commented it off first. 

Thank you

**Comment by alwinsonauyong** [comment]
commenting this out breaks the Gui but fixed public entries displaying properly, let me know once it's fixed so i can do the testing again~

**Comment by irvinlim** [comment]
It's supposed to wrap the JSON object so you don't need to return the `status` field when doing `res.status(200).json(returnObject)`. Fixed in 4f62597.
